siatest/renter/renter_test.go

```diff
@@ -716,6 +717,76 @@ func TestRenewFailing(t *testing.T) {
 	}
 }
 
+// testRenterPersistData checks if the RenterSettings are persisted
+func testRenterPersistData(t *testing.T, tg *siatest.TestGroup) {
+	// Grab the first of the group's renters
+	r := tg.Renters()[0]
+
+	// Check Settings, should be defaults
+	rg, err := r.RenterGet()
+	if err != nil {
+		t.Fatal(err, "Could not get Renter through RenterGet()")
+	}
+	// Current defaults are 2,0,0 for StreamCacheSize, MaxDownloadSpeed
+	// and MaxUploadSpeed
+	if rg.Settings.StreamCacheSize != 2 {
```

Comment:
you should be able to just compare to the default constants right, instead of having hardcoded values?